include/find_embedding/util.hpp

```diff
@@ -94,10 +94,10 @@ class LocalInteraction {
   public:
     virtual ~LocalInteraction() {}
     //! Print a message through the local output method
-    void displayOutput(int loglevel, const string& msg) const { displayOutputImpl(loglevel, msg); }
+    void displayOutput(int loglevel, const string& msg) const { displayOutputImpl(msg); }
```

Comment:
Please do not remove `loglevel` from these functions, as it's used elsewhere in the library.  It would be better to add this parameter into the `MyCppInteractions` overrides and ignore it there.